src/leveldb/port/port_posix.cc

```diff
@@ -55,7 +55,8 @@ void InitOnce(OnceType* once, void (*initializer)()) {
 
 bool HasAcceleratedCRC32C() {
 #if (defined(__x86_64__) || defined(__i386__)) && defined(__GNUC__)
-  unsigned int eax, ebx, ecx, edx;
+  unsigned int eax, ebx, edx;
+  unsigned int ecx = 0;
```

Comment:
This one should probably be fixed (or is fixed) upstream. Then we need to pull the latest version of leveldb subtree.

Reply:
:ok_hand: Removed the edit to leveldb